apps/web/components/icons/EventSeaLogo.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
```

Comment:
let's move all icons, images to `public` dir. to serve as static assets. Not required to create components for icons 
https://nextjs.org/docs/pages/building-your-application/optimizing/static-assets

Reply:
@meowyx

Reply:
oh yes, lemme do this. @chin-flags